src/redux/actionCreators.js

```diff
@@ -53,3 +54,14 @@ export const dismissActionsFromQueue = (
   type: actionTypes.DISMISS_ACTIONS_FROM_QUEUE,
   payload: actionTrigger,
 });
+
+export const queueSemaphoreChange = (
+  shouldHaltQueue: boolean,
+): FluxActionForQueueSemaphoreChange => ({
+  type: actionTypes.QUEUE_SEMAPHORE_CHANGE,
+  payload: shouldHaltQueue,
+});
+
+export default {
```

Comment:
Nice! 💯 Exporting action creators is something the community asked for some time ago. Would you mind extending it to export all of them?

Reply:
Good point, I'll do it in another commit 👍

Reply:
Done 👍